src/frontend/src/redux/system/actions.ts

```diff
@@ -19,10 +19,15 @@ export function logIn(email: string, password: string): any {
   };
 }
 
-export function logOut() {
-  removeCookie();
-  return {
-    type: LOG_OUT
+export function logOut(): any {
+  return (dispatch: Dispatch) => {
+    return apiService(dispatch, {
+      url: '/api/logout',
+      method: 'post'
+    }).then((response: any) => {
+      removeCookie();
+      dispatch({ type: LOG_OUT });
+    });
```

Comment:
Need to add a catch block, even if it just throws an alert to let the user know the request failed.

Reply:
Thanks for the catch; I've added an issue to improve from the default `alert`: https://github.com/codeforpdx/recordexpungPDX/issues/538.